### DISCUSSION TITLE
Always visible scroll bar in reader mode please

### DISCUSSION THREAD

**Discussion by BrilliantThings** [plateaukao/einkbro#409] [General]
I know it's fashionable to have the scroll progress indicator disappear, but I'd much prefer if it stayed visible permanently. Especially in reader mode. It's particularly invisible when using touch areas to scroll. 

Are you able to add this as an option? (Or am I not seeing it in settings?)

**Comment by plateaukao** [comment]
@BrilliantThings 
What's the purpose of showing scroll progress indicator?
If the purpose is to show current progress, you can add a page count info on toolbar. It serves the sam purpose.
As for adding visible scroll bar, since I don't use it, this may not be implemented in near future.

**Reply by BrilliantThings** [reply]
The purpose is a visual indication of the progress through the current article. But page numbers in the toolbar might serve the same purpose. Where is the setting to turn this on? I can't find it.

**Comment by plateaukao** [comment]
you can click on toolbar setting button (bottom row) on function menu.

**Reply by BrilliantThings** [reply]
That's perfect!
